usaspending_api/etl/helpers.py

```diff
@@ -107,3 +107,15 @@ def parse_numeric_value(string):
         return float(string)
     except:
         return None
+
+
+def get_fiscal_quarter(fiscal_reporting_period):
+    """Return the fiscal quarter."""
+    if fiscal_reporting_period in [1, 2, 3]:
+        return 1
+    elif fiscal_reporting_period in [4, 5, 6]:
+        return 2
+    elif fiscal_reporting_period in [7, 8, 9]:
+        return 3
+    elif fiscal_reporting_period in [10, 11, 12]:
+        return 4
```

Comment:
Yes, thanks for double checking that! Yes, there periods are already stored in the broker as federal fiscal periods, so 1 = October. I added a note to the docstring to clarify.